File: Java/TravelModel/src/main/java/travel/model/Employee.java
package travel.model;

import java.io.Serializable;
import java.util.Objects;

public class Employee implements Identifiable<Integer>, Serializable {
    private Integer ID;
    private String username;
    private String password;

    public Employee(Integer ID, String username, String password) {
        this.ID = ID;
        this.username = username;
        this.password = password;
    }

    public Employee(String username, String password) {
        this.ID = ID;
        this.username = username;
        this.password = password;
    }

    public String getUsername() {
        return username;
    }

    public void setUsername(String username) {
        this.username = username;
    }

    public String getPassword() {
        return password;
    }

    public void setPassword(String password) {
        this.password = password;
    }

    @Override
    public Integer getID() {
        return ID;
    }

    @Override
    public void setID(Integer id) {
        this.ID = id;
    }

    @Override
    public boolean equals(Object o) {
        if (this == o) return true;
        if (o == null || getClass() != o.getClass()) return false;
        Employee employee = (Employee) o;
        return Objects.equals(ID, employee.ID) && Objects.equals(username, employee.username) && Objects.equals(password, employee.password);
    }

    @Override
    public int hashCode() {
        return Objects.hash(ID, username, password);
    }

    @Override
    public String toString() {
        return "Employee{" +
                "ID=" + ID +
                ", username='" + username + '\'' +
                ", password='" + password + '\'' +
                '}';
    }
}

File: ProiectCS/travelModel/Employee.cs
using System;

namespace travelModel
{
    [Serializable]
    public class Employee : IIdentifiable<int>
    {
        public int ID { set; get; }
        public string Username { set; get; }

        public string Password { set; get; }

        public Employee(int ID, string username, string password)
        {
            this.ID = ID;
            Username = username;
            Password = password;
        }
        
        public Employee(string username, string password)
        {
            Username = username;
            Password = password;
        }

        public override string ToString()
        {
            return ID + ";" + Username + ";" + Password;
        }

        public override bool Equals(object obj)
        {
            Employee employee = obj as Employee;
            if (employee == null)
                return false;
            return ID == employee.ID;
        }

        public override int GetHashCode()
        {
            return base.GetHashCode();
        }
    }
}